scripts/histmakers/w_z_gen_dists.py

```diff
@@ -122,20 +140,20 @@ def build_graph(df, dataset):
         results.append(df.HistoBoost("nominal_Mll", [axis_Mll], ["ewMll", "nominal_weight"], storage=hist.storage.Weight()))
         axis_Mlly = hist.axis.Regular(100, 50, 150, name = "Mlly")
         results.append(df.HistoBoost("nominal_Mlly", [axis_Mlly], ["ewMlly", "nominal_weight"], storage=hist.storage.Weight()))
-
-    nominal_gen = df.HistoBoost("nominal_gen", nominal_axes, [*nominal_cols, "nominal_weight"], storage=hist.storage.Double())
+    nominal_gen = df.HistoBoost("nominal_gen", nominal_axes, [*nominal_cols, "nominal_weight"],  storage=hist.storage.Weight()) #storage=hist.storage.Double())
```

Comment:
The change of storage type may be already part of the upstream code, if you pull you should see it already